### ISSUE TITLE
Error: EISDIR: illegal operation on a directory

### ISSUE THREAD

**Issue by lngramos** [sasstools/sass-lint#555]
Running `sass-lint .` fails on my system, throwing the following error:

```
fs.js:603
  var r = binding.read(fd, buffer, offset, length, position);
                  ^

Error: EISDIR: illegal operation on a directory, read
    at Error (native)
    at Object.fs.readSync (fs.js:603:19)
    at Object.fs.readFileSync (fs.js:438:24)
    at /Users/myuser/projects/my-project/node_modules/sass-lint/index.js:180:18
    at Array.forEach (native)
    at Function.sassLint.lintFiles (/Users/myuser/projects/my-project/node_modules/sass-lint/index.js:178:9)
    at detectPattern (/Users/myuser/projects/my-project/node_modules/sass-lint/bin/sass-lint.js:16:18)
    at /Users/myuser/projects/my-project/node_modules/sass-lint/bin/sass-lint.js:91:5
    at Array.forEach (native)
    at Object.<anonymous> (/Users/myuser/projects/my-project/node_modules/sass-lint/bin/sass-lint.js:90:16)
```

I'm running Node v4.2.4 and the latest version of sass-lint (v1.5.1) on OS X 10.11 and fails with and without default config files.

**Comment by DanPurdy** [comment]
How're you running sass-lint? From the CLI? If so which command did you use?

**Comment by lngramos** [comment]
Yeah, from the CLI. Simply running `sass-lint .` as mentioned above.

I've also attempted running it via an npm script using the same command.

**Comment by nene** [comment]
I ran into the same thing.

Coming from scss-lint I assumed that sass-lint would scan a directory for .sass & .scss files. But apparently it doesn't - it just blindly attempts to read the directory as a file.

Workaround: `sass-lint 'path/to/**/*.scss'`

**Comment by DanPurdy** [comment]
Maybe we weren't so specific but it does mention the globbing and including of a file extension in the docs for the CLI and config file include section.

This probably should change though. Thanks :+1:

**Comment by mischkl** [comment]
This is happening to me despite having configured the include section in my sass-lint.yml file and specifying the config file. When I specify the SCSS glob pattern in the CLI directly, on the other hand, it simply does nothing, even though I know there are errors. So basically the CLI is completely broken for me, no idea why...

**Comment by DanPurdy** [comment]
I found the cause of this, if running sass-lint without any options the default glob would match folders labelled like the following 'o.scss' which happens the appear in the sass-graph package which is a dependency of node-sass. If you dont pass options to sass-lint the chances are it attempted to lint your node modules and came across this.

The fix should be with you for 1.8.0
